Objects/tupleobject.c

```diff
@@ -548,7 +548,7 @@ Raises ValueError if the value is not present.
 
 static PyObject *
 tuple_index_impl(PyTupleObject *self, PyObject *value, Py_ssize_t start,
-                 Py_ssize_t stop)
+                 Py_ssize_t stop, PyObject *key)
 /*[clinic end generated code: output=07b6f9f3cb5c33eb input=fb39e9874a21fe3f]*/
```

Comment:
Please, read the https://devguide.python.org/development-tools/clinic/#argument-clinic.

Reply:
Thank you!